README.md

```
@@ -134,34 +152,68 @@ For more details, run `holmes investigate <source> --help`
 HolmesGPT can investigate many issues out of the box, with no customization or training. Optionally, you can extend Holmes to improve results:
 
 **Custom Data Sources**: Add data sources (toolsets) to improve investigations
-   - If using Robusta SaaS: See [Robusta's docs](https://docs.robusta.dev/master/configuration/holmesgpt/custom_toolsets.html)
+   - If using Robusta SaaS: See [here](https://holmesgpt.dev/data-sources/custom-toolsets/)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)